audio-worklet/src/bitcrusher-demo.js

```diff
@@ -68,6 +57,16 @@ class BitcrusherDemo {
       this.reductionMin_ = this.paramReduction_.minValue;
     }
 
+    // The script processor will have a delay proportional to the size of its
+    // buffersize, which can be heard as a glitch as the user switches node.
```

Comment:
Don't see how this comment about glitching is relevant in any way to the buffer size variable. I think it would be more appropriate to explain this in the text for the demo.  So the user knows a glitch is possible.